learningresources/tests/test_api.py

```diff
@@ -55,6 +57,10 @@ def test_create(self):
             create_course(**self.kwargs)
         self.assertTrue(course_count() == after)
 
+        # Assert that passing None causes a NOT NULL constraint error
+        with self.assertRaises(IntegrityError):
```

Comment:
You can use `with self.assertRaises(IntegrityError) as ex` and check for the NOT NULL message in `ex.args` to make it more explicit, and not need the comment.  [example](https://github.com/mitodl/lore/blob/master/importer/tests/test_import.py#L75)